front/src/applications/stdcmV2/components/StdcmOrigin.tsx

```diff
@@ -77,10 +95,16 @@ const StdcmOrigin = ({ setCurrentSimulationInputs, disabled = false }: StdcmConf
               label={t('trainPath.time')}
               id="originTime"
               onChange={onOriginTimeInputChange}
+              isInvalid={!isNil(error)}
               value={originTime}
               disabled={disabled}
             />
           </div>
+          {!isNil(error) && (
```

Comment:
I think `isNil` is not compulsory here
```suggestion
          {error && (
```